ext/afform/core/ang/af/afRecaptcha.html

```diff
@@ -0,0 +1,5 @@
+<script src="https://www.google.com/recaptcha/api.js" async defer></script>
+<div class="crm-af-field">
```

Comment:
This div doesn't seem to belong here. This isn't a field so it just seems confusing. Is it needed for some reason?